Makefile

```diff
@@ -94,21 +94,21 @@ test:
 
 ## Run smoketest
 smoketest:
-	$(DOCKER_RUN) make release hugo/build test HUGO_URL=/ HUGO_CONFIG=test.toml HUGO_PUBLISH_DIR=test HTMLTEST_CONFIG=.htmltest.smoketest.yaml
+	$(DOCKER_RUN) make release hugo/build test HUGO_URL=/ HUGO_CONFIG=test.yaml HUGO_PUBLISH_DIR=test HTMLTEST_CONFIG=.htmltest.smoketest.yaml
 
 ## Generate a release config
+release: DOCKER_RUN_FLAGS += -e HUGO_CONFIG -e HTLMTEST_CONFIG -e HUGO_URL -e HUGO_PUBLISH_DIR -e HUGO_EDIT_URL
 release:
-	@[ "$(HUGO_CONFIG)" != "config.toml" ] || (echo "Cannot release with $(HUGO_CONFIG)"; exit 1)
+	@[ "$(HUGO_CONFIG)" != "config.yaml" ] || (echo "Cannot release with $(HUGO_CONFIG)"; exit 1)
 	@[ "$(HTMLTEST_CONFIG)" != ".htmltest.yml" ] || (echo "Cannot release with $(HTMLTEST_CONFIG)"; exit 1)
-	cat config.toml | \
-		sed 's,^baseURL.*,baseURL = "$(HUGO_URL)",' | \
-		sed 's,^publishDir.*,publishDir = "$(HUGO_PUBLISH_DIR)",' | \
-		sed 's,^editURL.*,editURL = "$(HUGO_EDIT_URL)",' \
-		> $(HUGO_CONFIG)
+	if [ "$(yq eval 'select(.editURL)' config.yaml)" ]; then \
```

Comment:
Note this is not run in a docker run

Reply:
Why do we need this conditional?

Reply:
I hadn't looked atthe `config.toml` file closely enough and thought that `config.yaml` shuld have a top-level field called `editURL`, which it doesn't. In reality, the yaml field is `params.editURL` and it is there. So, long chain of reasoning, but none of it is really applicable, it turns out.

It's maybe worth noting, though,  that [the version yq we're using](https://mikefarah.gitbook.io/yq/) doesn't have any `if...else...` functionality that I could find, which is why I ended up implementing the if,else in bash.

Reply:
(I simplified this in the most recent commit.)